ferrocene/doc/user-manual/src/rustc/cli.rst

```diff
@@ -33,12 +34,8 @@
 
          $ rustc -A warnings -A absolute-paths-not-starting-with-crate
 
-   .. cli:option:: -C
```

Comment:
Seems like this should be marked `unqualified` not removed?

Reply:
`-C` and `--codegen` are not actually flags. They are just a prefix for the codegen flags like `-C opt-level=3`. In the documentation we have been treating the codegen flags as their own, so adding a standalone `-C`/`--codegen` is not really useful IMO.